.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+> Provide a brief description about **why** this PR is necessary. Be sure to provide context.
+
+## Related issues and/or PRs
+
+> If this PR addresses or references any issues and/or other PRs, list them here.
+
+## Changes made
+
+> Outline the specific changes made in this pull request. Include relevant details, such as added features, bug fixes, code refactoring, or improvements.
+
+## How these changes were tested
+
+> Describe the testing that you have performed to ensure the changes are working as expected. Include details about the test cases or scenarios covered.
+
+## Checklist
+
+> The following is a best-effort checklist. If any items in this checklist are not applicable to this PR or are dependent on other, unmerged PRs, please still mark the checkboxes after you have read and understood each item.
+
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have updated the documentation to reflect the changes.
+- [ ] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
+- [ ] Tests (unit, integration, etc.) have been added for the changes.
+- [ ] My changes generate no new warnings.
+- [ ] Any dependent changes in other PRs have been merged and published.
+
+## Additional notes (optional)
+
+> Provide any additional information or notes that may be relevant to the reviewers or stakeholders.
+
+## Release notes
+
+> Provide release note text for this PR based on the guidelines at [Guidelines for Writing Release Notes](https://developers.scalar-labs.com/docs/style-guide/release-notes/).
+> 
+> For example:
+> 
+> Added validation for the Consensus Commit mutation operation.
```

Comment:
```suggestion
```

We don't have any release notes for `scalar-admin`.
So, I think we can remove this section at this time.
However, if we have any plans to create release notes in the future, I think it's OK to remain this section.

@supl 
What do you think?

Reply:
@kota2and3kan 
@josh-wong 

Thank you for pointing out this concern.

scalar-admin currently is a tool and library that is used by Scalar projects.
- scalar
- scalardb
- scalar-manager
- scalar-admin-k8s

It is not directly used by users so we don't have release notes (this saves maintenance costs).

In this assumption, I agree with Yokoyama-san. If we would just leave `N/A` to the `Release notes` section most of the time, then maybe it's easier to just remove it.

We can add this section back anytime whenever we need to provide release notes for some reason.

Reply:
Got it! That makes sense. I've removed this section.

Reply:
@supl 
Thank you for your answer!

> If we would just leave N/A to the Release notes section most of the time, then maybe it's easier to just remove it.
> We can add this section back anytime whenever we need to provide release notes for some reason.

I agree. Thank you!